### ISSUE TITLE
no-undef-class with :global

### ISSUE THREAD

**Issue by eddiemoore** [atfzl/eslint-plugin-css-modules#14]
If you have a global class that you only want something applied when it is a child of another class.
```css
/* fails */
.holder :global .Select {}
```

If `.holder` doesn't have a class of it's own, it says that `.holder` is unused. To get it to pass we need to change `.holder` to have css applied otherwise it has an error of `no-undef-class` with `.holder`

```css
/* passes */
.holder { display: block; }
.holder :global .Select {}
```

**Comment by atfzl** [comment]
It is happening because i am removing all the selectors with pseudoclass `:global`.

To get around this problem just use it like this:

```css
.holder {
  :global .Select {}
}
```

**Comment by Mike-Now** [comment]
Your fix cannot solve the following case:

```
:global(.Select) {
  .holder {}
}
```
However, there is workaround:
```
.holder {}
:global(.Select) {
  .holder {}
}
```

**Comment by jch254** [comment]
Is there any updates on this? The only workaround that worked for my use case was to define empty classes for all of the classes that have a :global selector as a child. The nested suggestion does not work with vanilla CSS.
